src/cascadia/TerminalSettingsEditor/Appearances.cpp

```diff
@@ -42,6 +43,145 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         return _hasPowerlineCharacters.value_or(false);
     }
 
+    Windows::Foundation::Collections::IMap<winrt::hstring, winrt::hstring> Font::FontAxesTagsAndNames()
+    {
+        auto fontAxesTagsAndNames = winrt::single_threaded_map<winrt::hstring, winrt::hstring>();
```

Comment:
this could at least start by building a `std::map` internally, then `std::move` the map into a `single_threaded_map` as the return value. That way you at least get performance while constructing the map.